### PR TITLE
Fix test adapter pipeline workflow

### PR DESCRIPTION
**Pull request by AugustasV**
**Description**

This PR fixes manual test adapter workflow

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/test_adaptersv2.yaml [hunk 1]
```diff
@@ -54,28 +54,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s_version: ['v1.20.11', 'v1.21.5', 'v1.22.2','v1.23.9']
-    runs-on: ubuntu-latest
+        k8s_version: ['v1.20.11', 'v1.21.5', 'v1.22.2','v1.23.9','v1.25.2']
```
Comment (leecalcote) [inline .github/workflows/test_adaptersv2.yaml:57]: Yay!

**Comment by leecalcote** [comment]
@Revolyssup feedback? @asubedy @hershd23

File: .github/workflows/test_adaptersv2.yaml [hunk 2]
```diff
@@ -148,17 +154,15 @@ jobs:
...
-      - name: Download patternfile
-        uses: actions/download-artifact@v2
-        with:
-          name: patternfile
       - name: Wait for Meshery pods to come up 
         run: sleep 100  
       # This pattern will try to install  service mesh(the first attempt can fail currently fot k8s componets so we will wait 100 seconds after the first apply and then apply again. Since pattern apply is idempotent, this wont be an issue.)
       - name: Pattern apply 
         run: |
+          wget https://raw.githubusercontent.com/service-mesh-patterns/service-mesh-patterns/master/samples/${{ inputs.patternfile_name }}
```
Comment (Zanda256) [inline .github/workflows/test_adaptersv2.yaml:162]: Why are we getting this pattern file? And where do we use it? Am I missing something?

Comment (AugustasV) [reply .github/workflows/test_adaptersv2.yaml:162]: @Zanda256 we have to apply patternfile. I added this line to download sample pattern file, according to patternfile_name

File: .github/workflows/test_adaptersv2.yaml [hunk 3]
```diff
@@ -148,17 +154,15 @@ jobs:
           echo "this is $svcip"
           config='{"contexts":{"local":{"endpoint":'$svcip',"token":"Default","platform":"kubernetes","adapters":[],"channel":"stable","version":"latest"}},"current-context":"local","tokens":[{"location":"auth.json","name":"Default"}]}'
           echo $config  > ~/.meshery/config.yaml
-      - name: Download patternfile
```
Comment (AugustasV) [inline .github/workflows/test_adaptersv2.yaml:151]: This block failed. How does it work? from where it downloads patternfile?

File: .github/workflows/test_adaptersv2.yaml [hunk 4]
```diff
@@ -54,28 +54,34 @@ jobs:
...
-        uses: manusa/actions-setup-minikube@v2.4.1
+        uses: manusa/actions-setup-minikube@v2.7.0
         with:
-          minikube version: "v1.23.2"
-          kubernetes version: '"${{ matrix.k8s_version }}"'
+          minikube version: "v1.27.1"
+          kubernetes version: '${{ matrix.k8s_version }}'
           driver: docker
-          start args: --cni "${{ inputs.k8s_cni }}"
+          start args: --cni bridge
```
Comment (Revolyssup) [inline .github/workflows/test_adaptersv2.yaml:74]: @AugustasV  Why hardcode it here?

File: .github/workflows/test_adaptersv2.yaml (hunk 3 above)
Comment (Revolyssup) [reply .github/workflows/test_adaptersv2.yaml:151]: This downloads the patternfile which is uploaded before applying this job in the `SetPatternfileAction`

**Comment by Chadha93** [comment]
@AugustasV Merge conflilcts...

**Comment by Chadha93** [comment]
@AugustasV Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by leecalcote** [comment]
@AugustasV to sign-off on that/those commits, there are a couple git commands to run --> https://github.com/meshery/meshery/pull/6365/checks?check_run_id=9314658641

**Comment by AugustasV** [comment]
#6498 for PR merge

**Pull request closed without merging**